htdocs/accountancy/class/accountancyexport.class.php

```diff
@@ -1460,13 +1462,13 @@ public function exportLDCompta10($objectLines)
 			print $date_lim_reglement.$separator;
 			// CNPI
 			if ($line->doc_type == 'supplier_invoice') {
-				if (($line->debit - $line->credit) > 0) {
+				if (($line->amount) < 0) {
```

Comment:
line->amount is not always filled. It was the field used in past but was replaced with debit and credit and is now not always filled.
Can you try to find a fix that use $line->debit, $line->credit and $line->sens